tests/integ/modin/conftest.py

```diff
@@ -346,14 +352,18 @@ def datetime_series(nper=30, freq: Frequency = "B", name=None) -> pandas.Series:
 def iloc_snowpark_pandas_input_map():
     return {
         "categorical[int]": pd.Categorical([1, 3, 4]),
-        "Index": pd.Index([-0.9, -1.0, -1.1, 0.0, 1.0, 0.9, 1.1, 1]),
+        # TODO: SNOW-1372242: Remove instances of to_pandas when lazy index is implemented
+        "Index": pd.Index([-0.9, -1.0, -1.1, 0.0, 1.0, 0.9, 1.1, 1]).to_pandas(),
```

Comment:
why we cannot keep it as pd.Index?

Reply:
because we pass it to a function that uses it as an iloc parameter, we need a native pd index

Reply:
Fixed this up just now